securedrop/journalist_templates/submission_preferences_saved_flash.html

```diff
@@ -0,0 +1,10 @@
+{# these are flashed messages for the logo upload file verifiaction #}
```

Comment:
let's update or remove this comment (these are the flashed messages for the submission file preferences)